internal/collector/otelcol.tmpl

```diff
@@ -4,14 +4,24 @@ receivers:
     collection_interval: {{ .Receivers.HostMetrics.CollectionInterval }}
     initial_delay: {{ .Receivers.HostMetrics.InitialDelay }}
     scrapers:
+      {{- if .Receivers.HostMetrics.Scrapers }}
+      {{- if .Receivers.HostMetrics.Scrapers.CPU }}
       cpu:
+      {{- end }}
+      {{- if .Receivers.HostMetrics.Scrapers.Disk }}
       disk:
-      load:
+      {{- end }}
+      {{- if .Receivers.HostMetrics.Scrapers.Filesystem }}
       filesystem:
+      {{- end }}
+      {{- if .Receivers.HostMetrics.Scrapers.Memory }}
       memory:
+      {{- end }}
+      {{- if .Receivers.HostMetrics.Scrapers.Network }}
       network:
-      paging:
-{{- end }}
+      {{- end }}
+      {{- end }}
+    {{- end }}  
```

Comment:
Indentation is a little off
```suggestion
{{- end }}  
```

Reply:
Will check if putting a yaml linter could help us catching these early.